README.md

```diff
@@ -0,0 +1,92 @@
+### BOOKSTORE
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+## About the Project
+
+> This project is built with React, it helps organise and save your favorite books.
+
+## Built With
+
+- React.js
+- Jest
+- Gitflow
+
+### Key Features
+
+- Page Setup
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://saluumaa.github.io/book-store/)
+
+
+## Getting Started
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-prefer-folder
+  git clone https://github.com/saluumaa/book-store
+```
+
+### Run this command to run server
+
+> npm start
+
+### Run this command build server
+
+> npm run build
+
+### Run this command to run tests
+
+> npm run test
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Salma Ibrahim**
+
+- GitHub: [@Saluumaa](https://github.com/saluumaa)
+- Twitter: [@SalmaHIbrahim20](https://twitter.com/SalmaHIbrahim20)
+- LinkedIn: [Salma ibrahim](https://www.linkedin.com/in/salma-ibrahim-78bb5a14a/)
+
+## 🔭 Future Features
+
+- 3 sub-pages
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/saluumaa/book-store/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 🙏 Acknowledgments
+
+I would like to thank everyone!
+
+## 📝 License
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Your MIT license [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) currently leads to an error page. I have attached a screenshot below. I think this might be because you don't have a license file in your project. So I will suggest that you include an MIT license file in your project. You can use this link to learn how to add one to your project :wink:

![image](https://user-images.githubusercontent.com/102699958/230039111-166dae37-6ff5-485f-b586-36043f34a1b3.png)

Reply:
Thank you, I fixed all the issues.